src/components/EpicButtons.test.tsx

```diff
@@ -40,4 +40,31 @@ describe('EpicButtons', () => {
 
         expect(queryByTestId('epic-buttons')).toBeNull();
     });
+
+    it('Does not render the reminder button when secondary CTA present', () => {
+        const variant = {
+            ...testData.content,
+            name: 'Example',
+            showTicker: false,
+            cta: {
+                text: 'Button text!',
+                baseUrl: 'https://support.theguardian.com/support',
+            },
+            secondaryCta: {
+                text: 'Secondary button!',
+                baseUrl: 'https://support.theguardian.com/support',
+            },
+            showReminderFields: {
+                reminderCTA: 'Reminder button!',
+                reminderDateAsString: 'June',
+                reminderDate: '2020-06-01T09:30:00',
+            },
+        };
+        const tracking = testData.tracking;
+
+        const { queryByText } = render(<EpicButtons variant={variant} tracking={tracking} />);
+
+        expect(screen.getByText('Secondary button!')).toBeInTheDocument();
+        expect(queryByText('Reminder button!')).toBeNull();
+    });
 });
```

Comment:
What do you think about an extra test to check the opposite of this? i.e. that the Reminder button was rendered when it's set in the JSON and there's no secondary CTA. I'm just aware I didn't add any tests for that logic when I worked on reminders, and maybe I should have. I'm happy to follow up with a separate PR if that's easier

Reply:
:+1: great idea, thanks

Reply:
Added in 24c6173